src/lib.rs

```diff
@@ -2595,11 +2595,22 @@ impl Build {
     }
 
     fn prefix_for_target(&self, target: &str) -> Option<String> {
+        // Put aside RUSTC_LINKER's prefix to be used as last resort
+        let rustc_linker = self.getenv("RUSTC_LINKER").unwrap_or("".to_string());
+        // let linker_prefix = rustc_linker.strip_suffix("-gcc"); // >=1.45.0
+        let linker_prefix = if rustc_linker.len() > 4 {
```

Comment:
I'd probably implement this as `if rustc_linker.ends_with("-gcc") { ... }`, but I don't really think this needs to be done here (I probably will be making a case for bumping `cc-rs`'s MSRV in coherence with whatever we decide in https://github.com/rust-lang/libs-team/issues/72, so I don't think this needs to be bad for that long).